static/src/components/builder/store/modules/table.js

```diff
@@ -5,15 +5,12 @@ const prop = (value, isVariable) => {
     return { value,
         isVariable }
 }
-export const getColumnObject = () => {
+export const getColumnObject = (id) => {
     return {
         name: '',
         header: '',
         component: 'plain-text',
-        filterable: false,
-        sortable: false,
-        isColumnId: false,
-        hideColumnId: false
+        id: 'Columns ' + id
```

Comment:
you can use template strings

Reply:
yes, and also replaced all of them by templating